src/Symfony/Component/PropertyInfo/Tests/Extractor/PhpDocExtractorTest.php

```diff
@@ -43,6 +44,19 @@ public function testExtract($property, ?array $type, $shortDescription, $longDes
         $this->assertSame($longDescription, $this->extractor->getLongDescription('Symfony\Component\PropertyInfo\Tests\Fixtures\Dummy', $property));
     }
 
+    public function testGetDocBlock()
+    {
+        $docBlock = $this->extractor->getDocBlock('Symfony\Component\PropertyInfo\Tests\Fixtures\Dummy', 'g');
```

Comment:
```suggestion
        $docBlock = $this->extractor->getDocBlock(Dummy::class, 'g');
```
Same for the other ones in this file

Reply:
Thank you!